app/controllers/api/v1/articles_controller.rb

```diff
@@ -3,4 +3,14 @@ def index
     articles = Article.all
     render json: articles, each_serializer: ArticlesIndexSerializer
   end
+
+  def show
+    begin
+      article = Article.find(params[:id])
+      render json: article, serializer: ShowArticleSerializer
+      
+    rescue => error
+      render json: { error: 'Article does not exist' }, :status => 404
```

Comment:
don use hash rockets for setting the status code

`status: :not_found`

Reply:
We asked Emma about this and she approved it